src/to_mdast.rs

```diff
@@ -1080,6 +1080,12 @@ fn on_exit_raw_text(context: &mut CompileContext) -> Result<(), message::Message
         }
     }
 
+    let value_bytes = value.as_bytes();
+    if value.len() > 2 && value_bytes[0] == b' ' && value_bytes[value.len() - 1] == b' ' {
```

Comment:
oh, actually -- I need to ensure that the value isn't _all_ space chars. If it is, they should not be stripped.